src/empty/empty-jumpbox-resources.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
Just out of curiousity what prompted the bump?

Reply:
The visual studio code ARM template extension prompted me 🙂